clang/test/CodeGenCoroutines/coro-symmetric-transfer-04.cpp

```diff
@@ -0,0 +1,65 @@
+// This tests that the symmetric transfer at the final suspend point could happen successfully.
```

Comment:
nit: I feel slightly better to add a link to the github page.

Reply:
Done.